modules/metrics/mule-metrics-exporter-api/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.mule.runtime</groupId>
+        <artifactId>mule-module-metrics-parent</artifactId>
+        <version>4.6.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>mule-metrics-exporter-api</artifactId>
+    <version>4.6.0-SNAPSHOT</version>
+    <name>Mule Metrics Exporter Api</name>
+    <description>Definitions for Mule Metrics Exporter Api</description>
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-api</artifactId>
+            <version>${opentelemetryVersion}</version>
+            <scope>compile</scope>
```

Comment:
remove compile as it is the default scope. (seeking where we did this in other parts of the code, not in this PR).